ndcube/ndcube.py

```diff
@@ -33,6 +33,37 @@ class NDCubeABC(astropy.nddata.NDData, metaclass=NDCubeMetaClass):
     def dimensions(self):
         pass
 
+    @abc.abstractproperty
+    def crop(self, lower_corner, upper_corner, coords_set='wcs', crop_by_values=False):
```

Comment:
I think I would rather have a `crop_by_values` method than a kwarg here. I think it's more inline with the world axis coords methods, and the methods on the WCS objects.

Reply:
I suspected you would say that.  But the option of cropping by values, whatever the API, is something you think we should offer users?

Reply:
I can see the benefit as far as API completeness goes and some people would probably find it useful. That being said, I worry about diluting the benefits of our Coordinates + quantities first API.